docs/BasicCommunication/OnEventChanged/index.md

```diff
@@ -57,45 +57,47 @@ When SDL receives an `OnEventChanged(DEACTIVATE_HMI, isActive: true)` from the H
+    - If there is a navigation app that is in (LIMITED, AUDIBLE) due to an active AUDIO_SOURCE event, and SDL receives `SDL.ActivateApp(appID_of_navigation_app)`, then SDL will change the navigation app's state to (FULL, AUDIBLE).
+    - If SDL receives `OnEventChanged(EMBEDDED_NAVI, isActive=true)`, SDL changes any media app in (LIMITED, AUDIBLE) to (LIMITED, ATTENUATED). After the EMBEDDED_NAVI event ends, SDL changes the media app's state to (LIMITED, AUDIBLE).
 !!!
 
-#### HMI Status of apps when AUDIO_SOURCE or EMBEDDED_NAVI event is activated
-|appHMIType|Event|hmiLevel before|audioStreamingState before|hmiLevel after|audioStreamingState after|
-|:---------|:----|:--------------|:-------------------------|:-------------|:------------------------|
-|Media|AUDIO_SOURCE|FULL/LIMITED|AUDIBLE|BACKGROUND|NOT_AUDIBLE|
-|Navigation|AUDIO_SOURCE|FULL/LIMITED|AUDIBLE|AUDIO_SOURCE|LIMITED|AUDIBLE|
-|Non-media|AUDIO_SOURCE|FULL/LIMITED|AUDIBLE|AUDIO_SOURCE|BACKGROUND|NOT_AUDIBLE|
-|Media|EMBEDDED_NAVI|FULL/LIMITED|AUDIBLE|LIMITED|AUDIBLE|
-|Navigation|EMBEDDED_NAVI|FULL/LIMITED|AUDIBLE|EMBEDDED_NAVI|BACKGROUND|NOT_AUDIBLE|
-|Non-media|EMBEDDED_NAVI|FULL/LIMITED|AUDIBLE|EMBEDDED_NAVI|BACKGROUND|NOT_AUDIBLE|
-
-#### Activating apps during active AUDIO_SOURCE or EMBEDDED_NAVI event
-|appHMIType|Event|New hmiLevel|New audioStreamingState|Keep event active|
-|:---------|:----|:-------|:------------------|:-------------------|
-|Media|AUDIO_SOURCE|FULL|AUDIBLE|false|
-|Navigation|AUDIO_SOURCE|FULL|AUDIBLE|true|
-|Non-media|AUDIO_SOURCE|FULL|NOT_AUDIBLE|true|
-|Media|EMBEDDED_NAVI|FULL|AUDIBLE|true|
-|Navigation|EMBEDDED_NAVI|FULL|AUDIBLE|false|
-|Non-media|EMBEDDED_NAVI|FULL|NOT_AUDIBLE|true|
+##### HMI Status of apps when AUDIO_SOURCE or EMBEDDED_NAVI event is activated
+|appHMIType|Event|HMI State before|HMI State after|
+|:---------|:----|:---------------|:--------------|
+|Media|AUDIO_SOURCE|(FULL/LIMITED, AUDIBLE)|(BACKGROUND, NOT_AUDIBLE)|
+|Navigation|AUDIO_SOURCE|(FULL/LIMITED, AUDIBLE)|(LIMITED, AUDIBLE)|
+|Non-media|AUDIO_SOURCE|(FULL/LIMITED, AUDIBLE)|(BACKGROUND, NOT_AUDIBLE)|
+|Media|EMBEDDED_NAVI|(FULL/LIMITED, AUDIBLE)|(LIMITED, AUDIBLE)|
+|Navigation|EMBEDDED_NAVI|(FULL/LIMITED, AUDIBLE)|(BACKGROUND, NOT_AUDIBLE)|
+|Non-media|EMBEDDED_NAVI|(FULL/LIMITED, AUDIBLE)|(BACKGROUND, NOT_AUDIBLE)|
+
+##### Activating apps during active AUDIO_SOURCE or EMBEDDED_NAVI event
+|appHMIType|Event|New HMI State|Keep event active|
+|:---------|:----|:------------|:----------------|
+|Media|AUDIO_SOURCE|(FULL, AUDIBLE)|false|
+|Navigation|AUDIO_SOURCE|(FULL, AUDIBLE)|true|
+|Non-media|AUDIO_SOURCE|(FULL, NOT_AUDIBLE)|true|
+|Media|EMBEDDED_NAVI|(FULL, AUDIBLE)|true|
+|Navigation|EMBEDDED_NAVI|(FULL, AUDIBLE)|false|
+|Non-media|EMBEDDED_NAVI|(FULL, NOT_AUDIBLE)|true|
 
 #### Parameters
 
```

Comment:
Broken image link. Change `./assets/Multiple_apps_activation_during_active_embedded_navigation_r_audio_source.png` to `./assets/Multiple_apps_activation_during_active_embedded_navigation_or_audio_source.png`